package.json

```diff
@@ -18,10 +18,12 @@
     "version:update": "lerna run version:update",
     "compile": "lerna run compile",
     "test": "lerna run test",
+    "test:ci:changed": "lerna run test --since origin/main --parallel",
     "test:browser": "lerna run test:browser",
     "bootstrap": "lerna bootstrap --no-ci",
```

Comment:
can you add hoisting here too? Will likely greatly speed up local installs

Reply:
Do you mind if we do it later in a different PR?
I think we can benefit from gaining some confidence in this new config in the CI and then apply it to local installs once we see it does not cause any issues.

Reply:
For example, it requires npm7 users to turn on legacy mode for peer dependenccy which is a valid change in CI, but nontrivial on local setups.

Reply:
of course